x-pack/plugins/telemetry_collection_xpack/schema/xpack_plugins.json

```diff
@@ -1771,10 +1771,14 @@
         }
       }
     },
-    "fileUploadTelemetry": {
+    "fileUpload": {
```

Comment:
Can we merge both telemetry types? `fileUpload` and `fileUploadTelemetry`

Reply:
The next step of the process of migrating ML file_upload plugin will be to remove the `maps_file_upload` plugin. That will remove  will `fileUploadTelemetry` type. For the new file_upload plugin, I wanted to start with a clean namespace to avoid any problems processing telemetry for the old plugin.